File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        temp = [0, 0, 0]
        for val in nums:
            temp[val] += 1

        for i in range(0, temp[0]):
            nums[i] = 0
        for i in range(temp[0], temp[0]+temp[1]):
            nums[i] = 1
        for i in range(temp[0]+temp[1], temp[0]+temp[1]+temp[2]):
            nums[i] = 2
        

File: 0075-sort-colors/0075-sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
  const temp = [0, 0, 0]
  for (let i = 0; i < nums.length; i++) {
    temp[nums[i]]++
    nums[i] = 0
  }
  for (let i = temp[0]; i < temp[0] + temp[1]; i++) {
    nums[i] = 1
  }
  for (let i = temp[0] + temp[1]; i < nums.length; i++) {
    nums[i] = 2
  }
};